release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -148,6 +148,46 @@ int result = -9223372036854775808; // error: '9223372036854775808' is out of ran
   - `createEnumDeclarationNode`
   - `createClassDefinitionNode`
 
+- Restricted query expression outer join type-binding to be inferred as a nillable type 
+    Earlier, the type binding for outer join could be any type-descriptor but now its restricted to be 'var' so that it will be inferred as a nillable type.
+   
+```ballerina
+type User record {|
+    readonly int id;
+    string name;
+|};
+
+type Login record {|
+    int userId;
+    string time;
+|};
+
+public function main() {
+    Login[] logins = [
+        {userId: 6789, time: "20:10:23"},
+        {userId: 1234, time: "10:30:02"},
+        {userId: 3987, time: "12:05:00"}
+    ];
+    
+    table<User> key(id) users = table [
+            {id: 1234, name: "Keith"},
+            {id: 6789, name: "Anne"}
+        ];
+
+    string[] _ = from var login in logins
+        outer join var user in users 
+        on login.userId equals user?.id
+        select user?.name + ":" + login.time;
```

Comment:
This should be a error since this is `string?` + `string` right?

Reply:
Yes let me update